Sources/PackageGet.php

```diff
@@ -641,7 +641,7 @@ function PackageUpload()
 
 	// Setup the correct template, even though I'll admit we ain't downloading ;)
 	$context['sub_template'] = 'downloaded';
-
+	$allowext = array('.zip','.tgz','.gz');
```

Comment:
SMF prefers spaces after commas in arrays on one line between elements. `$allowext = array('.zip', '.tgz', '.gz');`